lib/sample_loader.rb

```diff
@@ -306,15 +306,26 @@ def load filename
     def samples_dir
       unless defined? @samples_dir
         @samples_dir = nil
-        base_dir = "#{File.dirname File.dirname __dir__}/"
-        cur_dir = Dir.getwd
-        if cur_dir.start_with? base_dir
-          gem_name = cur_dir[base_dir.length..].split("/").first
-          tentative_dir = File.join base_dir, gem_name, "samples"
-          @samples_dir = tentative_dir if ::File.directory? tentative_dir
-        end
+        pathname = Pathname.new Dir.getwd
```

Comment:
Need a `require "pathname"` at the top if you use the library.

Reply:
Yes. My bad.